### PR TITLE
Add multi chats support

### PR DESCRIPTION
**Pull request by TheBlackArroVV**
This pr adds support for multi chats

### CODE REVIEW

File: db/database.rb [hunk 1]
```diff
@@ -7,32 +7,33 @@ def initialize
     @users = @db[:users]
   end
 
-  def increase_karma(user_name)
-    user = users.where(user_name: user_name)
+  def increase_karma(user_name, chat_id)
+    user = users.where(user_name: user_name, chat_id: chat_id)
```
Comment (yashka713) [inline db/database.rb]: Does it work with users, which already exist and do not have `chat_id`?

**Review by yashka713 — DISMISSED** [review]

File: db/database.rb (hunk 1 above)
Comment (TheBlackArroVV) [reply db/database.rb]: Now, but for now it used only in one chat, so i planned to just make a sql query which will fix it

Comment (TheBlackArroVV) [reply db/database.rb]: But i can create a data migration for it or rake task
What do you think, @yashka713 ?

Comment (yashka713) [reply db/database.rb]: I work on it.
The best way is creating `command` for admin users.
By this command, the admin will update user fields.
For example, by replaying for the message with a command.

Screen Shot 2019-07-17 at 21 44 10

This allows you to get information about the user, check that command started by admin, and update user fields.
If you find a better way write it [here](https://github.com/TheBlackArroVV/super_karma_bot/issues/5)

**Pull request merged**